modelexpress_client/python/tests/test_vllm_loader.py

```
@@ -809,3 +811,94 @@ def __init__(self):
+            assert len(mx_root.handlers) == 1
+            assert mx_root.handlers[0] is handler
+
+            child = logging.getLogger("modelexpress.metadata")
+            assert child.getEffectiveLevel() == logging.DEBUG
+        finally:
+            self._cleanup(vllm_logger)
+
+    def test_no_duplicate_handlers_on_repeated_calls(self):
+        from modelexpress.vllm_loader import _configure_vllm_logging
+
+        vllm_logger, handler = self._simulate_vllm_enginecore_logging()
```

Ruff (0.15.9)

[warning] Unpacked variable `handler` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)